### PR TITLE
picard: new application icon

### PR DESCRIPTION
**Pull request by phw**
## Description

Use the current icon for MusicBrainz Picard, which was introduced in 2017 with the 1.4 release.

## Motivation and Context

An outdated icon was used.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [ ] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [ ] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](add_link_to_original_repository_location)
- [Open Issues](link_to_the_generic_location_of_open_issues) *Add the different issues underneath, and tick those that are fixed in this PR*
  - [ ] Issue 1 link
  - [ ] Issue 2 Link
- [ ] *Include the link to the opened PR that removes the package from the original location*
- [ ] The [migration guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: icons/picard.svg [hunk 1]
```diff
@@ -1,161 +1,142 @@
...
-  <path clip-rule="evenodd" d="M 166.01314,707.80235 L 166.59926,710.95475 L 203.4248,903.68363 L 204.06642,906.86316 C 213.15578,954.44329 231.66477,969.40949 273.63726,962.69249 L 822.52374,875.10747 C 870.54724,867.44073 884.09914,845.98252 873.11464,794.87805 L 872.36954,791.43893 L 804.69524,477.35118 L 803.97964,473.98616 L 780.16614,363.56172 L 741.60574,184.52094 L 718.29754,76.373589 C 707.87364,28.021029 687.35194,14.726629 641.70404,26.451239 L 118.7716,160.91292 C 78.668396,171.2012 67.332566,191.3855 75.968336,236.59461 L 95.293611,337.65919 L 146.19729,604.05014 L 166.01314,707.80235 z M 349.35974,218.12246 C 334.78854,191.85942 347.66074,162.30769 378.05201,152.07361 C 408.45818,141.8765 444.88684,154.86419 459.45804,181.12724 C 474.02934,207.39029 461.15704,236.94202 430.76568,247.17619 C 400.35967,257.37327 363.9311,244.3855 349.35974,218.12246 z" id="path6464" style="opacity:0.66535429000000001;fill:#000000;fill-opacity:1;fill-rule:eveno
```
Comment (AdmiringWorm) [inline icons/picard.svg]: Could you please remove the width and height here?
I know it is part of the official icon, but the only purpose of this is to force a a maximum width and height of the svg icon, and shouldn't really be necessary.

Comment (AdmiringWorm) [reply icons/picard.svg]: Since it shows on the wrong line in the review comment, I am talking about removing line 16 and 17 in the new icon svg.

Comment (phw) [reply icons/picard.svg]: Sure, makes sense. I updated it with the width and height removed.

**Comment by AdmiringWorm** [comment]
@phw thank you for your contribution.
The new icon will be part of the next version of the picard package.

**Pull request merged**